lib/bridge/IrcHandler.js

```diff
@@ -390,12 +392,18 @@ IrcHandler.prototype.onMode = Promise.coroutine(function*(req, server, channel,
 
     // redundant if statement currently but eases burden when adding
     // support for more modes
-    if (["k", "i"].indexOf(mode) !== -1) {
+    if (privateModes.indexOf(mode) !== -1) {
```

Comment:
I imagined the point of this line was to be able to add other lines later on. So rather than putting the different `+s` handling logic inside this if block, add a new one later?